internal/sleuth/metric_impact_source_resource_test.go

```diff
@@ -0,0 +1,255 @@
+package sleuth
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/tfversion"
+)
+
+func TestAccMetricImpactSourceResource_v6(t *testing.T) {
+	// tests are run in parallel both locally & on CI, so we need to generate a random name so slugs don't collide
+	randomStr := acctest.RandStringFromCharSet(5, acctest.CharSetAlpha)
+	projectString := fmt.Sprintf("Terraform test project %s", randomStr)
+	projectSlug := fmt.Sprintf("terraform-test-project-%s", randomStr)
+
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			/* Skipping v0.14 & 0.15 to avoid the following error:
+			   Provider "registry.terraform.io/hashicorp/sleuth" planned an invalid value
+			   for
+			   sleuth_metric_impact_source.terraform_acc_test_dd.manually_set_health_threshold:
+			   planned value cty.NumberFloatVal(99.99) does not match config value
+			   cty.MustParseNumberVal("99.99").
+
+				No issues here, just a SDK bug/blurp
+			*/
+			tfversion.SkipBetween(tfversion.Version0_14_0, tfversion.Version1_0_0),
```

Comment:
added ignore for this because it fails with error that is not an error and TF answer was just - welp, old SDK issues 🤷